website/src/app/views/about/members.js

```diff
@@ -71,6 +72,7 @@ import kyle from '../../../images/about/memberImages/kyle.jpg';
 import dang from '../../../images/about/memberImages/dang.jpg';
 import rachel from '../../../images/about/memberImages/rachel.jpg';
 
+
```

Comment:
Everything looks good! Just remove this blank line and then I can merge it into dev :)